### PR TITLE
JPN-584 Style for discussions widget error messages

### PR DESCRIPTION
**Pull request by perjg**
## Links

* https://wikia-inc.atlassian.net/browse/JPN-584

## Description

A style to use for showing error messages when the <discussions /> tag can not be rendered due to incorrect parameters (e.g. setting columns=foo, where it should be either 1 or 2)

## Reviewers

@rogatty @bkoval

### CODE REVIEW

File: front/main/app/styles/_settings.scss [hunk 1]
```diff
@@ -65,6 +65,7 @@ $discussion-post-actions-border-color: #e6e6e6;
 $discussion-error-color: #ed1c24;
 $discussion-submit-icon-color: #5ba200;
 $discussion-widget-heading-background: #f6f6f6;
+$discussion-widget-errorcolor: #f00;
```
Comment (rogatty) [inline front/main/app/styles/_settings.scss]: `$discussion-widget-error-color`?

**Comment by rogatty** [comment]
One minor thing, LGTM

**Comment by perjg** [comment]
@rogatty In context:

$discussion-widget-errorcolor: #f00;
$discussion-widget-linkcolor: #00f;

If I change one I should probably change both: discussion-widget-error-color and discussion-widget-link-color ok?

**Comment by rogatty** [comment]
Yup, all the other variables in this file use dasherized names.

**Comment by perjg** [comment]
@rogatty Changed both variables to use dasherized names.

**Comment by rogatty** [comment]
LGTM

File: front/main/app/styles/component/_discussion-widget.scss [hunk 2]
```diff
@@ -40,3 +40,7 @@ $article-negative-padding: -$article-horizontal-padding;
 		padding-top: 0;
 	}
 }
+
+.widget-discussions-error {
```
Comment (bkoval) [inline front/main/app/styles/component/_discussion-widget.scss]: Hey, it seems like you're using a mix of "discussion-widget" and "widget-discussions" - is that the same thing? :)

**Comment by bkoval** [comment]
LGTM

File: front/main/app/styles/component/_discussion-widget.scss (hunk 2 above)
Comment (perjg) [inline front/main/app/styles/component/_discussion-widget.scss]: Yeah, I used widget-discussions for everything initially, but used discussion-widget for the colors I added in _settings.scss to make it line up better with existing names there.

Comment (bkoval) [inline front/main/app/styles/component/_discussion-widget.scss]: Not a blocker, but I'd think about a refinement at some point - would be nice to have the names in SCSS correspond to the component name :)

Comment (perjg) [inline front/main/app/styles/component/_discussion-widget.scss]: Understood. We're discussing some design changes for how error messages are shown, which will probably be done as a separate task after initial release, so I'll look into cleaning up the naming then.

**Pull request merged**